src/lib/support/tests/ExtraPwTestMacros.h

```diff
@@ -68,3 +68,19 @@
     {                                                                                                                              \
         test_name();                                                                                                               \
     }
+
+/**
+ *  @def EXPECT_SUCCESS(expr)
```

Comment:
how about `EXPECT_CHIP_NO_ERROR` or `EXPECT_NO_ERROR`?

Otherwise you should use `IsSuccess` 

Do we really want these overloads here though? This saves 10 chars now (or 4 chars if we do `EXPECT_CHIP_NO_ERROR`) and adds cognitive load by having more macros.

I would rather use things that save more lines of code ... but 10 chars and no lines of code seems extreme.

Reply:
It's about readability, not about saving chars

When you have `EXPECT_EQ(really long line that goes all the way to the end or maybe goes on to the next line, CHIP_NO_ERROR)`, there is cognitive overload to find that you're expecting success, whereas `EXPECT_SUCCESS` immediately tells you that you expect a call to be successful

It also makes the refactoring easier - it's easier to wrap currently-ignored `ChipError` in `EXPECT_SUCCESS` vs having to also add a `CHIP_NO_ERROR` parameter

`EXPECT_SUCCESS` is in-line with the GoogleTest naming scheme, e.g. `EXPECT_TRUE`, as well as in-line with the macro naming, i.e. `SuccessOrExit`, `SuccessOrLog`, etc. - and does have the added benefit of saving a few chars